.env.development

```diff
@@ -24,4 +24,4 @@ APPLE_PAY_CURRENCY_CODE='USD',
 APPLE_PAY_START_SESSION_URL='http://localhost:18130/payment/cybersource/apple-pay/start-session/',
 APPLE_PAY_AUTHORIZE_URL='http://localhost:18130/payment/cybersource/apple-pay/authorize/',
 APPLE_PAY_SUPPORTED_NETWORKS='amex,discover,visa,masterCard',
-APPLE_PAY_MERCHANT_CAPABILITIES='supports3DS,supportsCredit,supportsDebit',
\ No newline at end of file
+APPLE_PAY_MERCHANT_CAPABILITIES='supports3DS,supportsCredit,supportsDebit',
```

Comment:
What was the change in this?

Reply:
It was just a line-ending change - there wasn't a blank line at the end of the file.